osm2gtfs/core/creator_factory.py

```diff
@@ -28,83 +28,143 @@ def __repr__(self):
 
     def get_agency_creator(self):
         selector = self.selector
+
+        # Define class name
+        if "_" in selector:
+            split_selector = selector.split("_")
+            class_name_selector = str()
+            for part in split_selector:
+                class_name_selector += part.capitalize()
+        else:
+            class_name_selector = selector.capitalize()
+
         try:
             module = importlib.import_module(
                 ".creators." + selector + ".agency_creator_" + selector,
                 package="osm2gtfs")
             agency_creator_override = getattr(
-                module, "AgencyCreator" + selector.capitalize())
-            print("Agency creator: " + selector.capitalize())
+                module, "AgencyCreator" + class_name_selector)
+            print("Agency creator: " + selector)
             return agency_creator_override(self.config)
         except ImportError:
             print("Agency creator: Default")
             return AgencyCreator(self.config)
 
     def get_feed_info_creator(self):
         selector = self.selector
+
+        # Define class name
+        if "_" in selector:
+            split_selector = selector.split("_")
+            class_name_selector = str()
+            for part in split_selector:
+                class_name_selector += part.capitalize()
+        else:
+            class_name_selector = selector.capitalize()
+
```

Comment:
can we avoid to duplicate the class_name_selector creation code in each function ?

Reply:
+1 That would be much nicer indeed!